simple-web-app/app/src/main/scriptsSql/create_tables.sql

```diff
@@ -0,0 +1,92 @@
+drop table usr;
+drop table adm;
+drop table coach ;
+drop table student;
```

Comment:
Тут лучше использовать `drop table if exists`
Последовательность таблиц должна быть такая, чтобы удаление не натыкалось на foreigh keys, а то usr не удалить первой, т.к. на неё ссылаются adm и т.д.
```suggestion
drop table if exists adm;
drop table if exists coach ;
drop table if exists student;
drop table if exists usr;
```

Reply:
Исправлено